tests/satosa/frontends/test_openid_connect.py

```diff
@@ -190,7 +190,7 @@ def test_provider_configuration_endpoint(self, context, frontend):
             "claims_supported": ["email"],
             "grant_types_supported": ["authorization_code", "implicit"],
             "issuer": BASE_URL,
-            "require_request_uri_registration": True,
+            "require_request_uri_registration": False,
```

Comment:
Latest `oic` changed the default value of `require_request_uri_registration` to be `False`.